src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/FilterClause.java

```diff
@@ -12,6 +20,112 @@
     example = """
              {"name": "Aaron", "country": "US"}
               """)
-public record FilterClause(LogicalExpression logicalExpression) {}
+public record FilterClause(LogicalExpression logicalExpression) {
+  public void validate(CommandContext commandContext) {
+    DocumentProjector indexingProjector = commandContext.indexingProjector();
+    // If nothing specified, everything indexed
+    if (indexingProjector.equals(DocumentProjector.identityProjector())) {
```

Comment:
I think `indexingProjector.isIdentityProjection()` would be better.

Reply:
`isIdentityProjection()` is a function inside the private class `PathCollector`. Do you want me to create a new function?

Reply:
Hmmh. `isIdentityProjection()` should be member of `DocumentProjector`... I'll check.